MapleServer2/Data/Static/MapEntityStorage.cs

```diff
@@ -115,6 +127,16 @@ public static List<CoordS> GetHealingSpot(int mapId)
             return healthSpot.GetValueOrDefault(mapId);
         }
 
+        public static PatrolData GetPatrolData(int mapId, string patrolDataName)
+        {
+            return PatrolDatas.GetValueOrDefault(mapId).FirstOrDefault(x => x.Name == patrolDataName);
+        }
```

Comment:
Now that you modified this class, you can change the method with one single return and use expression body.
```suggestion
        public static PatrolData GetPatrolData(int mapId, string patrolDataName) => PatrolDatas.GetValueOrDefault(mapId).FirstOrDefault(x => x.Name == patrolDataName);
```